src/Benchmarks/Data/ApplicationDbContext.cs

```diff
@@ -9,6 +9,6 @@ public class ApplicationDbContext : DbContext
     {
         public DbSet<World> World { get; set; }
 
-        public DbSet<Fortune> Fortunue { get; set; }
+        public DbSet<Fortune> Fortune { get; set; }
```

Comment:
Thanks. Guess I need to fix my dev env so I can actually build this stuff :cow: :boy: